assets/js/modules/thank-with-google/components/settings/SettingsView.js

```diff
@@ -93,14 +93,20 @@ export default function SettingsView() {
 		);
 	} else {
 		supporterWall = (
-			<p className="googlesitekit-settings-module__meta-item-data">
-				<Link
-					href={ supporterWallURL }
-					className="googlesitekit-settings-module__cta-button"
-				>
-					{ __( 'Add supporter wall', 'google-site-kit' ) }
-				</Link>
-			</p>
+			<Fragment>
+				<p className="googlesitekit-settings-module__meta-item-info">
+					A supporter wall widget shows the list of people who
+					supported your site using Thank with Google
```

Comment:
This text isn't translated, but needs to be rendered with `__()` so it can be translated.

Also, the text in the ACs is a full sentence so this should be too.

```suggestion
					{ __( 'A supporter wall widget shows the list of people who supported your site using Thank with Google.' ) }
```